python/example_robot_data/robots_loader.py

```diff
@@ -483,8 +483,16 @@ class DoublePendulumLoader(RobotLoader):
     urdf_subpath = "urdf"
 
 
-def loadDoublePendulum():
+class DoublePendulumContinuousLoader(RobotLoader):
+    path = "double_pendulum_description"
+    urdf_filename = "double_pendulum_continuous.urdf"
+    urdf_subpath = "urdf"
+
+
+def loadDoublePendulum(continuous=False):
```

Comment:
This load function is deprecated, will be removed, and shouldn't be used. Therefore, there is no need to update it

Reply:
Alright!